test/tools/libtesteth/ImportTest.cpp

```diff
@@ -289,11 +296,18 @@ json_spirit::mObject& ImportTest::makeAllFieldsHex(json_spirit::mObject& _o, boo
 			str = value.get_str();
 		else if (value.type() == json_spirit::array_type)
```

Comment:
In the current design, this function should not look inside arrays because `postParseJson()` does look into the array.